src/GPL-1.0.xml

```diff
@@ -0,0 +1,242 @@
+<spdx name="GNU General Public License v1.0 only" identifier="GPL-1.0" osi-approved="false">
+  <urls>
+    <url>http://www.gnu.org/licenses/old-licenses/gpl-1.0-standalone.html</url>
+  </urls>
+  <notes>This license was released: February 1989. This license has been deprecated. This refers to when this GPL 1.0
```

Comment:
> in this case "deprecated" means by the author in lieu of a more recent version

The [field docs][1] you reference just say:

> Only factual information should be included here (e.g. the license has been deprecated).

which doesn't say anything about “by the author”.  If we are tracking license-author advice, I recommend breaking it out into a new field:

    <authorSuggests>GPL-3.0</authorSuggests>

Although the rest of this ontology doesn't have much to say about who the license author is.  Maybe pull in some Dublin Core to define the [`creator`][2] whose advice we're passing along?

And if this is ongoing advice for every license, `creator` is probably too restrictive.  What if the original creator wants to hand off responsibility for providing advice to a third party?  Maybe that's not a use-case we care about covering?

[1]: https://spdx.org/spdx-license-list/license-list-overview#fields
[2]: http://dublincore.org/documents/2012/06/14/dcmi-terms/?v=terms#terms-creator